README.md

```diff
@@ -61,6 +61,8 @@ pip install -e .
 make install-dep
 make test  # just the unit tests
 tox  # all the code checks
+tox -e py39-pydocstyle # perform only pydocstyle tests (py39 is the version of the python interpreter you have installed)
```

Comment:
I would suggest a section on how to run a specific tox environment using this as an example. And how to list all of the configured tox environments (`tox -l` if I remember correctly)

@kinow Later you and I should synchronize a CONTRIBUTING.md across all the CWL code projects. I'm rather partial to one I worked on for k-h-mer: https://khmer.readthedocs.io/en/latest/dev/getting-started.html ( from https://khmer.readthedocs.io/en/latest/dev/ )

Reply:
>I would suggest a section on how to run a specific tox environment using this as an example. And how to list all of the configured tox environments (tox -l if I remember correctly)

+1

>@kinow Later you and I should synchronize a CONTRIBUTING.md across all the CWL code projects. I'm rather partial to one I worked on for k-h-mer: https://khmer.readthedocs.io/en/latest/dev/getting-started.html ( from https://khmer.readthedocs.io/en/latest/dev/ )

Great idea! During the video call I was actually looking for a CONTRIBUTING.md, before suggesting the README.md update. So I guess other developers would probably expect this useful building instructions to be somewhere in a CONTRIBUTING file.

And great job on the khmer file! It even includes multi-platform docs :clap: Let's do it!

Reply:
@Th3nn3ss just pending to address this part of @mr-c 's feedback

>I would suggest a section on how to run a specific tox environment using this as an example. And how to list all of the configured tox environments (tox -l if I remember correctly)

Then we should be ready to merge it :tada: